### PR TITLE
Run Typos in CI

### PR DESCRIPTION
**Pull request by szepeviktor**
Needs #1629 to succeed.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -208,3 +208,15 @@ jobs:
...
+    name: "Typos"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
+
+      - name: "Search for misspellings"
+        uses: "crate-ci/typos@master"
```
Comment (kukulich) [inline .github/workflows/build.yml:222]: Is it possible to use v1 or something like this?

Comment (szepeviktor) [reply .github/workflows/build.yml:222]: No, they don't have this kind of tag.
https://github.com/crate-ci/typos/tags
Typos is getting better at typos fast.

BTW I find their GHA action terrible.
Using this instead https://github.com/szepeviktor/byte-level-care/blob/f1c750a0424ab8a0b7e96cd253f8cb3973b324c6/.github/workflows/spelling.yml#L31-L50

Comment (kukulich) [reply .github/workflows/build.yml:222]: Ok, so I would use the latest tag v1.16.17

Comment (szepeviktor) [reply .github/workflows/build.yml]: Done 🍏

**Comment by kukulich** [comment]
Thanks.

**Comment by szepeviktor** [comment]
Glad to meet your bytes.

**Pull request merged**